apps/cognito-functions/package.json

```diff
@@ -4,8 +4,10 @@
   "private": true,
   "scripts": {
     "clean": "shx rm -rf dist/ out/",
-    "compile": "npm run clean && tsc",
-    "build": "npm run compile && esbuild dist/main.js --bundle --platform=node --outfile=out/main.js",
+    "precompile": "npm run clean",
+    "compile": "tsc",
+    "prebuild": "npm run compile",
```

Comment:
what's the need to run `tsc` before `esbuild` during deploys?

Reply:
you are right. we are going to fix it with #405